ui/appui-react/src/appui-react/uistate/useUiStateStorage.tsx

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module UiStateStorage
+ */
+
+import * as React from "react";
+import { LocalStateStorage, UiStateStorage } from "@itwin/core-react";
+import { UiFramework } from "../UiFramework";
+import { SyncUiEventArgs, SyncUiEventDispatcher, SyncUiEventId } from "../syncui/SyncUiEventDispatcher";
+
+/** eslint-disable deprecation/deprecation */
+
+/** @public */
+export function useUiStateStorageHandler(): UiStateStorage { // eslint-disable-line deprecation/deprecation
```

Comment:
Does this need the deprecation tag since UiStateStorage is not deprecated?

Reply:
Nope.  that one and a couple removed.